pkg/util/time.go

```diff
@@ -0,0 +1,35 @@
+package util
+
+import (
+	"errors"
+	"math/rand"
+	"time"
+)
+
+type TimeRange struct {
+	min   int
+	max   int
+	units time.Duration
+}
+
+func NewTimeRange(units time.Duration, min, max int) *TimeRange {
```

Comment:
as an aside, while I think you should merge this as is, because more files probably make sense... 

I think it'd be more clear to just do min/max durations and cut out the third argument... and I'd probably just do `max(0, r.Random())` to avoid the (logical) underflow rather than panic (potentially I think rather a long time after startup?)

Reply:
> I think it'd be more clear to just do min/max durations and cut out the third argument

Yeah, there's definitely some improvements that can be made. Going to leave that for #35.

> rather than panic (potentially I think rather a long time after startup?)

Currently, `TimeRange` is _mostly_ used in `Watch` calls, which are all done at startup. So: should be ok, for now.